### DISCUSSION TITLE
HealthKit 2.0 (Post your feedback here)

### DISCUSSION THREAD

**Discussion by stanpanchenkosommoio** [buildnatively/community#51] [Ideas]
### Use case

```Text
Water tracking (READ)
```

### Idea

```Text
Water tracking (READ)
```

**Comment by Justin-Schepke** [comment]
Please add the ability to READ "HKWorkoutActivityType"!! This is very important for User Experience in order to categorize and easily identify user activity data.

**Reply by romanfurman6** [reply]
We will add it for sure 🙌 in the future updates
